js/ui/editor/editor.js

```diff
@@ -178,107 +159,61 @@ const Editor = Widget.inherit({
         return false;
     },
 
-    _renderValidationState: function() {
-        const isValid = this.option('isValid') && this.option('validationStatus') !== VALIDATION_STATUS_INVALID;
-        const validationMessageMode = this.option('validationMessageMode');
-        const $element = this.$element();
+    _getValidationErrors: function() {
         let validationErrors = this.option('validationErrors');
         if(!validationErrors && this.option('validationError')) {
             validationErrors = [this.option('validationError')];
         }
-        $element.toggleClass(INVALID_CLASS, !isValid);
-        this.setAria(VALIDATION_STATUS_INVALID, !isValid || undefined);
-
-        if(!hasWindow()) {
-            return;
-        }
+        return validationErrors;
+    },
 
+    _removeValidationMessage: function() {
         if(this._$validationMessage) {
             this._$validationMessage.remove();
             this.setAria('describedby', null);
             this._$validationMessage = null;
         }
```

Comment:
Confusing name `removeValidationMessage` means that you are removing the `validationMessage` variable

Reply:
fixed